test/e2e/framework/pod/resource.go

```diff
@@ -506,9 +506,9 @@ func newExecPodSpec(ns, generateName string) *v1.Pod {
 			TerminationGracePeriodSeconds: &immediate,
 			Containers: []v1.Container{
 				{
-					Name:    "exec",
-					Image:   BusyBoxImage,
-					Command: []string{"sh", "-c", "trap exit TERM; while true; do sleep 5; done"},
+					Name:  "agnhost-pause",
+					Image: imageutils.GetE2EImage(imageutils.Agnhost),
```

Comment:
This won't work; `newExecPodSpec` is being used to create pods for `kubectl exec`-ing into, so it needs to have an image that has stuff in it

Reply:
oh, i see agnhost has some binaries in it... but I'm assuming that the reason why a whole bunch of tests failed on this PR is that it doesn't have everything needed...

Reply:
I'm troubleshooting failing E2Es. But effort is to use `agnhost`. agnhost:2.0 or further release holds various cli based app and system binaries. (https://github.com/kubernetes/kubernetes/pull/78389)

Reply:
This change looks valid.  Can we keep this and drop the change to `service.go`?

Reply:
Done